invenio_records_lom/resources/serializers/schemas/oai.py

```diff
@@ -164,7 +166,7 @@ def get_centity(self, entities):
 
         return [
             {
-                "vcard": e,  # TODO: fix vcard
+                "vcard": make_lom_vcard(fn=e),
```

Comment:
minor: could this whole return statement written in one line? Maybe by removing the ',' after make_lom_vcard?